src/Plugin.php

```diff
@@ -107,6 +107,9 @@ public static function init() {
     add_filter('script_loader_tag', __NAMESPACE__ . '\Asset::loader_tag', 10, 3);
     add_filter('style_loader_tag', __NAMESPACE__ . '\Asset::loader_tag', 10, 3);
 
+    // Filter whether to kill elasticpress sync during imports.
```

Comment:
```suggestion
    // Update Elasticsearch index during imports.
```

Reply:
I don't think that comment makes sense with the filter.
If I see  `// Update Elasticsearch index during imports.` and something returning false I would expect that it doesn't get updated.

Reply:
The comment does not need to repeat the double negation of the filter hook and its value.  Instead, the docs should explain as short and simple as possible what the customization is doing.

But if you prefer something else, maybe this works for you:

```suggestion
    // Prevent Elasticsearch becoming out of sync after importing data.
```